airbyte-integrations/connectors/source-aha/source_aha/aha.yaml

```diff
@@ -0,0 +1,73 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "{{ config['url'] }}/api/v1"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config['api_key'] }}"
+  increment_paginator:
```

Comment:
This is not used in the retriever. You must add this component to your base stream or retriever. Check other connectors how they do this.

Reply:
Fixed this. Thanks @marcosmarxm  for pointing it out

Reply:
I raised the issue  [19274](https://github.com/airbytehq/airbyte/issues/19274) while testing the connector
Please let me know if my observations are valid